src/main/java/Echo.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
+public class Echo {
+    public static void main(String[] arg){
+        String line;
+        Scanner in = new Scanner(System.in);
+        while(true) {
+            System.out.println(" ");
+            line = in.nextLine();
+            if (line.equalsIgnoreCase("bye")) {
+                Binks.createLineSpacing();
+                System.out.println("Bye. Hope to see you again soon!");
+                Binks.createLineSpacing();
+                break;
+            }
+            Binks.createLineSpacing();
```

Comment:
Statements under the same logic component have the same level of abstraction. Good job!